### ISSUE TITLE
Menu tables do not exist after upgrade

### ISSUE THREAD

**Issue by rallek** [zikula/core#3416]
| Q                 | A
| ----------------- | ---
| Zikula Version    | 1.4.6 ci68
| PHP Version       | 5.5.30

### Expected behavior
Menu Module should be avaialble

### Actual behavior
after upgrading from 1.4.4 to 1.4.6 ci 68 I opened the MenuModule in backend. The following stack Trace is telling my problem. If I look into the database I can not find any menu table.

````
[1] Doctrine\DBAL\Exception\TableNotFoundException: An exception occurred while executing 'SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db20.menu_items' doesn't exist
    at n/a
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php line 53

    at Doctrine\DBAL\Driver\AbstractMySQLDriver->convertException('An exception occurred while executing 'SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC':

SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db20.menu_items' doesn't exist', object(PDOException))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php line 128

    at Doctrine\DBAL\DBALException::driverExceptionDuringQuery(object(Driver), object(PDOException), 'SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC', array())
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 855

    at Doctrine\DBAL\Connection->executeQuery('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC', array(), array(), null)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleSelectExecutor.php line 50

    at Doctrine\ORM\Query\Exec\SingleSelectExecutor->execute(object(Connection), array(), array())
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 727

    at Doctrine\ORM\AbstractQuery->getResult()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity/Repository/NestedTreeRepository.php line 67

    at Gedmo\Tree\Entity\Repository\NestedTreeRepository->getRootNodes()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/system/MenuModule/Controller/MenuController.php line 45

    at Zikula\MenuModule\Controller\MenuController->listAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(MenuController), 'listAction'), array(object(Zikula_Request_Http)))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/index.php line 23

[2] Doctrine\DBAL\Driver\PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db20.menu_items' doesn't exist
    at n/a
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php line 106

    at Doctrine\DBAL\Driver\PDOConnection->query('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 852

    at Doctrine\DBAL\Connection->executeQuery('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC', array(), array(), null)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleSelectExecutor.php line 50

    at Doctrine\ORM\Query\Exec\SingleSelectExecutor->execute(object(Connection), array(), array())
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 727

    at Doctrine\ORM\AbstractQuery->getResult()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity/Repository/NestedTreeRepository.php line 67

    at Gedmo\Tree\Entity\Repository\NestedTreeRepository->getRootNodes()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/system/MenuModule/Controller/MenuController.php line 45

    at Zikula\MenuModule\Controller\MenuController->listAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(MenuController), 'listAction'), array(object(Zikula_Request_Http)))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/index.php line 23

[3] PDOException: SQLSTATE[42S02]: Base table or view not found: 1146 Table 'web16_db20.menu_items' doesn't exist
    at n/a
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php line 104

    at PDO->query('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php line 104

    at Doctrine\DBAL\Driver\PDOConnection->query('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php line 852

    at Doctrine\DBAL\Connection->executeQuery('SELECT m0_.id AS id_0, m0_.title AS title_1, m0_.lft AS lft_2, m0_.lvl AS lvl_3, m0_.rgt AS rgt_4, m0_.options AS options_5, m0_.root_id AS root_id_6, m0_.parent_id AS parent_id_7 FROM menu_items m0_ WHERE m0_.parent_id IS NULL ORDER BY m0_.lft ASC', array(), array(), null)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query/Exec/SingleSelectExecutor.php line 50

    at Doctrine\ORM\Query\Exec\SingleSelectExecutor->execute(object(Connection), array(), array())
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/Query.php line 321

    at Doctrine\ORM\Query->_doExecute()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 969

    at Doctrine\ORM\AbstractQuery->executeIgnoreQueryCache(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 924

    at Doctrine\ORM\AbstractQuery->execute(null, '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/doctrine/orm/lib/Doctrine/ORM/AbstractQuery.php line 727

    at Doctrine\ORM\AbstractQuery->getResult()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/gedmo/doctrine-extensions/lib/Gedmo/Tree/Entity/Repository/NestedTreeRepository.php line 67

    at Gedmo\Tree\Entity\Repository\NestedTreeRepository->getRootNodes()
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/system/MenuModule/Controller/MenuController.php line 45

    at Zikula\MenuModule\Controller\MenuController->listAction(object(Zikula_Request_Http))
        in  line 

    at call_user_func_array(array(object(MenuController), 'listAction'), array(object(Zikula_Request_Http)))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 144

    at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Zikula_Request_Http), '1')
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/HttpKernel.php line 64

    at Symfony\Component\HttpKernel\HttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php line 69

    at Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel->handle(object(Zikula_Request_Http), '1', true)
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/vendor/symfony/symfony/src/Symfony/Component/HttpKernel/Kernel.php line 185

    at Symfony\Component\HttpKernel\Kernel->handle(object(Zikula_Request_Http))
        in /home/www/www.oldtimer-ig-osnabrueck.de/test/Zikula_Core-1.4.6/index.php line 23
````

**Comment by rallek** [comment]
The Sytem/MenuModule is there and in extensions it is listed
![grafik](https://cloud.githubusercontent.com/assets/935250/22347485/7611f292-e408-11e6-982e-7d23f47ddcdb.png)

**Comment by craigh** [comment]
fixed by #3422 @rallek please retest-upgrade and confirm.

**Comment by rallek** [comment]
Just did an new upgrade from my 1.4.4 to the 1.4.6 ci.build 87
The upgrade went fine mainly. The upgrade told me something went wrong with the upgrade of the modules.

I have to activate all my modules and have to do some upgrades of system modules. Menu Module is still 1.0.0 with the same bahavior. Tables are missing.

Two other issues happens with Page Lock and security center. I will post two new issues with stack trace for that.

I did the upgrade with my german language pack for 1.4.6, But the routes.de.po is just a fresh copy of routes.tempate.po to avoid some issues by that.
